<?xml version="1.0" encoding="UTF-8"?>

<launch>
    <arg name="initial_x" default="-2" />
    <arg name="initial_y" default="-0.5" />
    <arg name="des_x" default="2" />
    <arg name="des_y" default="-3" />
    <param name="initial_x" value="$(arg initial_x)" />
    <param name="initial_y" value="$(arg initial_y)" />
    <param name="des_pos_x" value="$(arg des_x)" />
    <param name="des_pos_y" value="$(arg des_y)" />
    <node pkg="bug_alg" type="follow_wall_visbug21.py" name="wall_follower"/>
    <node pkg="bug_alg" type="go_to_point.py" name="go_to_point" />
    <node pkg="bug_alg" type="visbug21.py" name="bug0" output="screen" />
</launch>
